code/game/gamemodes/antag_spawner.dm

```diff
@@ -228,6 +228,13 @@
 	icon = 'icons/obj/wizard.dmi'
 	icon_state = "vial"
 
+	var/shatter_msg = "<span class='notice'>You shatter the bottle, no \
+		turning back now!</span>"
+	var/veil_msg = "<span class='warning'>You sense a dark presence lurking \
+		just beyond the veil...</span>"

```

Comment:
just curious as to why you even bother splitting these into two lines. they're not even that long